modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/display/context/TranslateDisplayContext.java

```diff
@@ -150,11 +151,12 @@ else if (infoFieldSetEntry instanceof InfoFieldSet) {
 				continue;
 			}
 
+			Stream<InfoField> infoFieldsStream = infoFields.stream();
```

Comment:
When there's only one stream var, you can call it `stream`, without specifying the item type.